README.md

```diff
@@ -13,13 +13,9 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
 ## Deployments
 
-### Shared Dev
-
-[https://experimenter-app.dev.mozaws.net/](https://experimenter-app.dev.mozaws.net/)
-
 ### Staging
 
-[https://experimenter.stage.mozaws.net/](https://experimenter.stage.mozaws.net/)
+[https://stage.experimenter.nonprod.dataops.mozgcp.net/](https://stage.experimenter.nonprod.dataops.mozgcp.net/)
```

Comment:
FWIW, if your link label+href are the same, you can usually just use the shortcut syntax:

```md
<https://stage.experimenter.nonprod.dataops.mozgcp.net/>
```

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links